pkg/local-storage/member/controller/volume_migrate_task_worker.go

```diff
@@ -278,6 +286,18 @@ func (m *manager) volumeMigrateSyncReplica(migrate *apisv1alpha1.LocalVolumeMigr
 		}
 	}
 
+	//Indicates lvm migration is complete
+	var anno map[string]string
+	if anno = vol.GetAnnotations(); anno == nil {
+		anno = make(map[string]string)
+	}
+	anno[apisv1alpha1.VolumeMigrateCompletedAnnoKey] = apisv1alpha1.MigrateFinish
+	vol.SetAnnotations(anno)
```

Comment:
this may cause error when migration happens on the volume later.

suggest change: remove this migrate anno after migration is done.

Reply:
`completed` anno should update until migration is finished totally in case of scheduling to the tobe removed node.

Reply:
> this may cause error when migration happens on the volume later.
> 
> suggest change: remove this migrate anno after migration is done.

may be no effect, ignore it.